exporter/collector/internal/normalization/disabled_normalizer.go

```diff
@@ -34,57 +34,41 @@ func NewDisabledNormalizer() Normalizer {
 type disabledNormalizer struct{}
 
 // NormalizeExponentialHistogramDataPoint returns the point without normalizing.
-func (d *disabledNormalizer) NormalizeExponentialHistogramDataPoint(point pmetric.ExponentialHistogramDataPoint, _ uint64) (pmetric.ExponentialHistogramDataPoint, bool) {
+func (d *disabledNormalizer) NormalizeExponentialHistogramDataPoint(point pmetric.ExponentialHistogramDataPoint, _ uint64) bool {
```

Comment:
I see a lot of these codecov warnings, but it seems like it should be covered by benchmark_test. Is it just flaking?

Reply:
We don't test the disabled normalizer (all of those are for the standard one).